Update Italian Translation(WIP)

Nutjob opened the pull request.

akuma06 commented on translations/it-it.all.json:
```diff
@@ -636,11 +716,707 @@
   {
     "id": "cancel",
     "translation": "Annulla"
   }
```
You miss a comma here :)

Nutjob commented:
I put WIP in the title because the translation it's still not complete (mostly mod panel and other account stuff), i was too busy the last few days so i just added all the missing stuff and corrected some more.
Sorry about the missing comma, i wasn't online at all yesterday.
